src/src/tx_main.cpp

```diff
@@ -640,13 +640,18 @@ static void CheckConfigChangePending()
 
 bool ICACHE_RAM_ATTR RXdoneISR(SX12xxDriverCommon::rx_status const status)
```

Comment:
I would leave this function as it was and just put your check at the front.
I'm pretty sure there was a reason why the flag was set to false after the packet was processed.